src/examples/basics.md

```diff
@@ -1 +1,120 @@
 # SML Basics Examples
+
+## Types
+For each of the following expressions, state its type and value.
```

Comment:
These are declarations, not expressions - we should either leave them all as expressions or write something like "For each of the following declarations, state the type and value of `x`."